WebUI/views/index.jade

```diff
@@ -44,29 +46,25 @@ block content
           .col-sm-9
             input#git_url.form-control(type='text', placeholder='Enter URL of Github Repository', name='git_url')
         .form-group
-          label.control-label.col-sm-3(for='author') Author:
-          .col-sm-9
-            input#author.form-control(type='text', placeholder='Enter Author', name='author')
-        .form-group
           label.control-label.col-sm-3(for='doc_theme') Doc Theme:
           .col-sm-9
-            input#doc_theme.form-control(type='text', placeholder='Enter Documentation Theme', name='doc_theme')
-        .form-group
-          label.control-label.col-sm-3(for='doc_path') Doc Path:
-          .col-sm-9
-            input#doc_path.form-control(type='text', placeholder='Enter Documentation Path', name='doc_path')
-        .form-group
-          label.control-label.col-sm-3(for='project_name') Project Name:
-          .col-sm-9
-            input#project_name.form-control(type='text', placeholder='Enter Project Name' , name='project_name')
-        .form-group
-          label.control-label.col-sm-3(for='version') Version:
-          .col-sm-9
-            input#version.form-control(type='text', placeholder='Enter Version', name='version')
+            select#doc_theme.form-control(type='text', placeholder='Enter Documentation Theme', name='doc_theme')
+             option(val='fossasia') fossasia
```

Comment:
I would also suggest to change the label to fossasia_theme since users using the travis interface or .yaydoc.yml would still need to type fossasia_theme and this might confuse them.